File: wxrtc/src/main/java/com/wx/rtc/rtc/PeerConnectionManager.java
package com.wx.rtc.rtc;

import com.wx.rtc.WXRTCDef;

public class PeerConnectionManager {
    public String userId;
    public String sendSdpUrl;
    public boolean needReconnect;
    public boolean videoRecvEnabled;
    public boolean videoRecvMute;
    public boolean audioRecvMute;
    public float audioVolume;
    public ProxyVideoSink videoSink;
    public PeerConnectionClient client;
    public WXRTCDef.WXRTCRenderParams renderParams;
}

File: wxrtc/src/main/java/com/wx/rtc/rtc/PeerConnectionManager.kt
package com.wx.rtc.rtc

import com.wx.rtc.WXRTCDef.WXRTCRenderParams

internal class PeerConnectionManager {
    @JvmField
    var userId: String? = null
    @JvmField
    var sendSdpUrl: String? = null
    @JvmField
    var needReconnect: Boolean = false
    @JvmField
    var videoRecvEnabled: Boolean = false
    @JvmField
    var videoRecvMute: Boolean = false
    @JvmField
    var audioRecvMute: Boolean = false
    @JvmField
    var audioVolume: Float = 0f
    @JvmField
    var videoSink: ProxyVideoSink? = null
    @JvmField
    var client: PeerConnectionClient? = null
    @JvmField
    var renderParams: WXRTCRenderParams? = null
}
